lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -110,7 +111,7 @@ def display_peek(self, dataset):
 
 class PlyAscii(Ply, data.Text):
 
-    file_ext = "plyascii"
+    file_ext = "ply"
```

Comment:
Thomas, for the time being can you leave the extensions as they are and use in paraview a if condition like
```
#if $hda.ext in ['plybinary', 'plyascii']:
   cp $hda foo.ply 
#end 
```